Resources/Prototypes/Recipes/Construction/Graphs/weapons/metal_hydrogen_axe.yml

```
@@ -0,0 +1,14 @@
+  id: MetalHydrogenAxeConstruct
+  start: start
+  graph:
+    - node: start
+      edges:
+      - to: icon
+        steps:
+        - material: MetalHydrogen
+          amount: 12
+          doAfter: 10
+    - node: icon
+      entity: MetalHydrogenAxe
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)